noxfile.py

```diff
@@ -0,0 +1,61 @@
+# mypy: disallow-untyped-defs=False, disallow-untyped-calls=False
+
+import glob
+import shutil
+
+import nox
+
+nox.options.sessions = ["lint"]
+nox.options.reuse_existing_virtualenvs = True
+
+
+@nox.session(python=["2.7", "3.4", "3.5", "3.6", "3.7", "3.8", "pypy", "pypy3"])
+def tests(session):
+    def coverage(*args):
+        session.run("python", "-m", "coverage", *args)
+
+    session.install("coverage<5.0.0", "pretend", "pytest", "pip>=9.0.2")
+
+    if "pypy" not in session.python:
+        coverage("run", "--source", "packaging/", "-m", "pytest", "--strict")
+        coverage("report", "-m", "--fail-under", "100")
+    else:
+        # Don't do coverage tracking for PyPy, since it's SLOW.
```

Comment:
This is in accordance to the commit message for tox's corresponding lines: fbc93678b687d28a35a79649887d01a62da548a3